drivers/SmartThings/matter-rvc/src/init.lua

```diff
@@ -162,6 +162,8 @@ local function rvc_run_mode_supported_mode_attr_handler(driver, device, ib, resp
   )
 
   local component = device.profile.components["runMode"]
+  local labels = get_field_labels_of_supported_modes(device, RVC_RUN_MODE_SUPPORTED_MODES)
+  device:emit_component_event(component, capabilities.mode.supportedArguments(labels))
```

Comment:
What is the difference between `supportedArguments` and `supportedModes`?

Also, should this be `labels_of_supported_modes` instead of just `labels`?